keyboards/leeku/finger65/finger65.h

```diff
@@ -0,0 +1,34 @@
+/* Copyright 2018 Rasmus Schults <rasmusx@gmail.com>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include "quantum.h"
+
+#define LAYOUT_65_ansi( \
+    K4A, K4B, K4C, K4D, K4E, K4F, K4G, K4H, K4I, K4J, K4K, K4L, K4M, K4N, K4O, K4P,\
+    K3A, K3B, K3C, K3D, K3E, K3F, K3G, K3H, K3I, K3J, K3K, K3L, K3M, K3N,      K3P,\
+    K2A, K2B, K2C, K2D, K2E, K2F, K2G, K2H, K2I, K2J, K2K, K2L, K2M, K2N,      K2P,\
+    K1A, K1B, K1C, K1D, K1E, K1F, K1G, K1H, K1I, K1J, K1K, K1L, K1M,      K1O, K1P,\
+    K0A, K0B, K0C, K0D,           K0G,      K0I, K0J,           K0M, K0N, K0O, K0P\
```

Comment:
This actually seems like too many keys for an ANSI 65% layout. If this contains all possible switch positions, it should be named `LAYOUT_all`, and a proper 65% layout added.

Reply:
done. your wish is ...

just kept a default 65% layout and kept the full layout commented out for future reference, should someone else want to add or change it

Reply:
If you want a split backspace then that would be an additional `LAYOUT_65_ansi_split_bs` ;)

Reply:
what do you mean? I see two layouts there? one with and one without split bs :-P

Reply:
🤔